### ISSUE TITLE
Problem: Archivematica job detail messages are not visible in the Enduro UI

### ISSUE THREAD

**Issue by fiver-watson** [artefactual-sdps/enduro#954]
**Describe the issue**

Currently, there is no way to see AM STDOUT and/or STDERR messages related to specific jobs run in Archivematica. This can make troubleshooting more difficult and provides operators with less insight into what actions are being performed on their content. 

**Expected behavior**

Currently, Enduro workflow tasks are organized into a table, with a Notes column. At present, that Notes column is not being used. 

For now, the initial implementation proposal is to add a "View" link to the Notes column. When clicked, this should open an expandable dropdown below the current row, that will include the STDOUT or STDERR messages from the related Archivematica job. 

**Screenshots**

A rough wireframe has been prepared below: 

![Enduro-AM-Notes-mockup-v0 1](https://github.com/artefactual-sdps/enduro/assets/5068277/6648a615-d882-4a87-9615-dd319771ece5)

You can see a bit more context on the following Miro board: 

* https://miro.com/app/board/uXjVKPpvKlM=/?moveToWidget=3458764590041643261&cot=14

Ideally, much like in AM, STDERR messages would be in a light red frame, rather than the light grey of the STDOUT as shown in the wireframe. 

**Additional context**

This issue is part of an incremental effort to improve the end user messages shown in workflow tasks. Part 1, hiding some of the unnecessary AM tasks, is described in issue #950. Future tickets will also be filed to customize the task names and outcome messages shown in Enduro, for better clarity to operators.

**Comment by jraddaoui** [comment]
We should be careful about accessibility, I'm not sure how that expandable section inside a table (inside another expandable section) will work in that regard.

**Comment by aseles13** [comment]
I am not sure how much work or analysis is still needed to be able to include this in PoC#2, but if we can show them some error messages (user or system) it would be a good step forward.

**Comment by mcantelon** [comment]
Could we, to avoid complication and possible accessibility issues, open job details in a new browser tab?

**Comment by djjuhasz** [comment]
I don't think that the Archivematica job task STDOUT/STDERR data are available through the Archivematica API.  The [AM Task API](https://www.archivematica.org/en/docs/archivematica-1.17/dev-manual/api/api-reference-archivematica/#task) only returns a status code and the timing information in the API response. :(

**Comment by fiver-watson** [comment]
A couple of notes on this issue: 

1) the proposed UI design has changed since this issue was first filed - as per issue #1077 , we have moved from a table-based list of workflow tasks, to card based ones
2) we have also learned that the tool output from AM microservices (e.g. things that JHOVE or sigfried etc spit out - the STDOUT) is not available in the AM API. Meaning we can't show as much detail as anticipated. 
3) Nevertheless, we do  want to show a bit more information about the outcome of tasks. In the following spreadsheet, we did include proposed success messages for AM microservice tasks - see: 

* https://docs.google.com/spreadsheets/d/10jByMHs6kTl4B22VCnwlEd2tLPb5tdJ595wZg-J96ts/edit?gid=1083719011#gid=1083719011

One option is that this ticket is repurposed to include better general outcome messages, whether success or failure, for AM tasks.

**Comment by scollazo** [comment]
Be aware that stdout/stderr messages  (and everything else related to a transfer) are removed from Archivematica dashboard database periodically, as they are considered transient data.

**Comment by sallain** [comment]
@scollazo good info, thanks. I think we're shifting towards thinking of the SIP/Ingest workflow overall as transient data, so I think that'll be okay.

**Comment by sallain** [comment]
Next step: I will file an issue in the Archivematica Issues repo with an Archivematica-focused version of the spreadsheet and we'll discuss with the Maintainers.

**Comment by sallain** [comment]
After a few chats with different people, I found [this issue](https://github.com/archivematica/Issues/issues/625#issuecomment-2783966915) in the AM Issues repo and decided that it suited our use case enough that I could reuse it. We'll see where it goes from here!
